workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,380 @@
+Shall we begin?
+
+```python
+from turtle import *
+from Figures import *
+from random import randint
+
+```
+Here, we import all fuctions from 'turtle' library, all functions from 'Figures' file (which we just created) and finally 'randint' function from 'random' library so that we can use it to generate a random integer.
+Why do we need a random integer? We'll see .
+
+
+```python
+speed(0)
+
+bg = turtle.Screen()
+bg.bgcolor("#99004D") #Setting back ground color
+
+y = -100
+width = 240
+
+```
+`speed()` function determines the speed of the turtle, ie, the speed in which the figures get drawn on the screen. When we set that to zero it moves as fast as it can. 
+Next, we set the background color to maroon using hexcode (#99004D).
+Find the right value for ******y******, through trial and error. Set some random value first and then, iterate to find the right value which brings your turtle to the center.
+For ******width******, provide any value you desire. 
+
+```python
+# Constructing the tree
+box1(turtle, "#4F4553", -15, y - 40, 30, 40) #Stem of the tree
+```
+Now we make a make a stem for the tree by passing desired values for each parameter. Color will be a shade of brown (#4F4553) and the rest as you see above. 
+![alt tree_stem_completed](https://cloud-4ipo9natc.vercel.app/4image3.png) 
+
+
+Next thing we are gonna make is the tree leaves with tiny balls on either sides.
+It is as easy as it sounds. See if the code is understandable.
+
+```python
+while width > 20 :
+    width = width - 20
+    height = randint(20,30)
+    x = 0 - width/2
+    box1(turtle, "#05A167", x, y, width, height) # Box for the leaves of tree
+    balls(turtle, "#fc045b", x, y, 5) # Decorative balls
+    balls(turtle, "#fc045b", -x, y, 5) # Decorative balls
+    y = y + height
+
+```
+Not too difficult, is it? Let's break this down.
+
+```python
+width = width - 20
+height = randint(20,30)
+```
+In each iteration, the width of the next rectangle gets reduced to a smaller size so that in the end we get a triangle kind of look for the entire leaf part (green portion). We assign a random value for the height as that helps to give the tree an irregular shape rather than a giving same height for all the rectangles. If you would like, you could reduce the width by a random integer too. 
+
+```python
+x = 0 - width/2
+```
+This is just to set the starting 'x' coordinate.
+
+```python
+box1(turtle, "#05A167", x, y, width, height) # Box for the leaves of tree
+```
+Now we draw one complete green color filled box for first layer of leaves.
+![alt One_layer_of_leaves](https://cloud-4ipo9natc.vercel.app/5image4.png) 
+
+```python
+balls(turtle, "#fc045b", x, y, 5) # Decorative balls
+```
+This statement draws a small ball of radius 5 with a shade of red (#fc045b) filled in it. 
+![alt one_ball_completed](https://cloud-4ipo9natc.vercel.app/6image5.png) 
+Now we need one more tiny ball on the right end of the christmas tree, right? 
```

Comment:
Add a blank line between the image and this line. Markdown reads 2 lines directly under each other as one line, so it’s trying to put this text on the same line as the image right now and it looks weird.